### PR TITLE
Change all enum types to integer

### PR DESCRIPTION
**Pull request by indocomsoft**
If you notice, all those with enum columnns will always store `nil`.

According to http://api.rubyonrails.org/v5.1/classes/ActiveRecord/Enum.html
> the values map to integers in the database

So I converted all those enum to be of integer type instead

### CODE REVIEW

File: app/models/availability.rb [hunk 1]
```diff
@@ -6,18 +6,16 @@
...
 #  created_at    :datetime         not null
 #  updated_at    :datetime         not null
 #  status        :boolean          not null
+#  day           :integer
 #
 # Indexes
 #
-#  index_availabilities_on_time_range_id                      (time_range_id)
-#  index_availabilities_on_user_id                            (user_id)
-#  index_availabilities_on_user_id_and_time_range_id_and_day
```
Comment (donjar) [inline app/models/availability.rb]: Did you not have this index in your db?

**Comment by indocomsoft** [comment]
My bad. I changed my migration so that it wouldn't delete the index.

File: app/models/availability.rb (hunk 1 above)
Comment (indocomsoft) [reply app/models/availability.rb]: Yeah :/ I realised what I did wrong. Fixed!

File: db/migrate/20180329164621_change_all_enum_to_integer.rb [hunk 2]
```diff
@@ -0,0 +1,6 @@
+class ChangeAllEnumToInteger < ActiveRecord::Migration[5.1]
+  def change
+    remove_column :timeslots, :day
```
Comment (donjar) [inline db/migrate/20180329164621_change_all_enum_to_integer.rb]: Next time you can just use `change_column`

Comment (indocomsoft) [reply db/migrate/20180329164621_change_all_enum_to_integer.rb]: I did but there was some error with casting :/

**Pull request merged**